setup.py

```diff
@@ -46,12 +46,14 @@
     ],
     'test': [
         "hypothesis==3.44.26",
+        "py-solc==3.1.*",
         "pytest~=3.2",
         "pytest-asyncio==0.8.0",
         "pytest-cov==2.5.1",
         "pytest-xdist==1.18.1",
         "pytest-watch>=4.1.0,<5",
-        "termcolor==1.1.*"
+        "termcolor==1.1.*",
+        "web3>=4.1.0,<5.0.0",
```

Comment:
py-solc and web3 dependencies can go into the tox dependency list for py36-benchmark, to keep the general test requirements shorter.

Reply:
Good point. However, moving them somewhere into the tox setup makes them very subtle though. It should be most obvious for someone new to the code base how to install these dependencies. So how about I introduce a new `deps['benchmark']` section that is not included in `dev` and then make sure to install that one only for the benchmark CI run?